src/protocols/secure_channel/tests/TestPASESession.cpp

```diff
@@ -45,31 +73,19 @@ class TestSecurePairingDelegate : public SessionEstablishmentDelegate
     uint32_t mNumPairingComplete = 0;
 };
 
-class TestSecurePairingExchangeContext : public Messaging::ExchangeContext
+class MockAppDelegate : public ExchangeDelegate
 {
 public:
-    CHIP_ERROR SendMessage(Protocols::Id protocolId, uint8_t msgType, System::PacketBufferHandle msgPayload,
-                           const Messaging::SendFlags & sendFlags) override
+    void OnMessageReceived(ExchangeContext * ec, const PacketHeader & packetHeader, const PayloadHeader & payloadHeader,
+                           System::PacketBufferHandle buffer) override
     {
-        mNumMessageSend++;
-        ReturnErrorOnFailure(mMessageSendError);
-        if (peer != nullptr)
-        {
-            PayloadHeader payloadHeader;
-            payloadHeader.SetMessageType(protocolId, msgType);
-            peer->OnMessageReceived(peerContext, PacketHeader(), payloadHeader, std::move(msgPayload));
-        }
-
-        return CHIP_NO_ERROR;
+        IsOnMessageReceivedCalled = true;
+        ec->Close();
     }
 
-    void Free() override {}
-
-    uint32_t mNumMessageSend     = 0;
-    CHIP_ERROR mMessageSendError = CHIP_NO_ERROR;
+    void OnResponseTimeout(ExchangeContext * ec) override {}
 
-    PASESession * peer                       = nullptr;
-    Messaging::ExchangeContext * peerContext = nullptr;
+    bool IsOnMessageReceivedCalled = false;
```

Comment:
nit: Why did the naming style change here?

Reply:
`MockAppDelegate` class was copied from other unit tests that are using `MessagingContext`. And, `TestSecurePairingExchangeContext` was removed. It happened to be in the same place in code, and appears that the naming style was changed on purpose. I'll update the copied code to match the current code style.